test/io.jl

```diff
@@ -242,4 +242,49 @@ end
     end
 end
 
+@testset "eltypes tests" begin
+    df = DataFrame(A = 1:3, B = ["x", "y", "z"])
+    df.A = map(x -> Int32(x), df[:A])
+
+    io = IOBuffer()
+    show(io, MIME("text/plain"), df, eltypes=true)
+    str = String(take!(io))
+    @test str == "3×2 DataFrame\n│ Row │ A     │ B      │\n│     │ Int32 │ String │\n├─────┼───────┼────────┤\n│ 1   │ 1     │ x      │\n│ 2   │ 2     │ y      │\n│ 3   │ 3     │ z      │"
```

Comment:
better wrap tests in `""""` as in other places in test files- this is more readable and the lines are not so long.

Reply:
changed in 4bb6529

Reply:
I was not clear what I meant. I proposed to write something like (taken from other tests in show.jl):
```
    io = IOContext(IOBuffer(), :displaysize=>(11,40), :limit=>true)
    show(io, df_big)
    str = String(take!(io.io))
    @test str == """
    25×5 DataFrame. Omitted printing of 2 columns
    │ Row │ x1       │ x2       │ x3       │
    │     │ Int64    │ Int64    │ Int64    │
    ├─────┼──────────┼──────────┼──────────┤
    │ 1   │ 11569985 │ 12178109 │ 17315979 │
    ⋮
    │ 24  │ 14660095 │ 13529407 │ 19204569 │
    │ 25  │ 16992761 │ 15379139 │ 13043102 │"""
```
In this way you immediately visually see what is the expected output (+ please add the tests for other display types). Thank you!

Reply:
@bkamins Ah sorry that makes sense! Put in a commit a9d733f to make the string more readable